utils/kafka-test-utils/src/main/scala/pl/touk/nussknacker/engine/kafka/EmbeddedKafkaServer.scala

```diff
@@ -106,7 +108,8 @@ object EmbeddedKafkaServer {
 class EmbeddedKafkaServer(
     zooKeeperServerOpt: Option[(NIOServerCnxnFactory, ZooKeeperServer)],
     val kafkaServer: Server,
-    val kafkaAddress: String
+    val kafkaAddress: String,
+    val tempDir: File
```

Comment:
Should I also make `kafkaServer` private: It doesn't have any methods that are useful externally:
![obraz](https://github.com/TouK/nussknacker/assets/1409807/ddef31fe-069d-42d2-ba14-04b67c338b1d)

Reply:
I thought that we keep here `KafkaServer` instead of `Server`. If nobody has problem with this, it means that there are two options: 1) nobody used it 2) somebody did a downcast. In both situations IMO it is ok to hide it. In case of 2) we change the type and reexpose it.